format_ideas/gaussian1d-option-2.yaml

```diff
@@ -0,0 +1,56 @@
+# Option 2: Allow simplified representation for minimal models
+
+
+!transform/tags/gaussian1d-1.0
+  name: Minimal single model
+
+  parameters:
```

Comment:
Would a mixture of simple cases and more complex ones be permitted? Some listed under this parameters attribute and some with a parameters tag?

Reply:
Yes that shouldn't be a problem

Reply:
I'd like for all parameters (including those with the tag) to be nested under `parameters`.  I think that'll let us handle most of the models with the same converter class, since the implementation can just initialize the model with whatever it finds under `parameters`.